src/styles/blocks/catalog.scss

```
@@ -17,9 +17,14 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
-
+    transition: transform $time, color $time;
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    &:hover {
+      transform: scale(1.2) ;
+      color: #34568b;
```

create variables for colors

```suggestion
      color: $c-someColor;
```